CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v2.2.1
+
+* Use SHA256 algorithm instead of MD5
```

Comment:
I just realized we're not following the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention 😦.  What do you think about updating this entry to something like:
```suggestion
### Changed
* Use SHA256 algorithm instead of MD5 for public key fingerprints.
```